### PR TITLE
Add comments for buildFromReviseRatingProperties

### PR DESCRIPTION
**Pull request by belyrs**

### CODE REVIEW

**Review by amarts — APPROVED** [review]

File: packages/network-score/src/Scoring.ts [hunk 1]
```diff
@@ -458,10 +455,42 @@ export async function buildFromRevokeRatingProperties(
...
+ * It takes a rating entry, chain space URI, and author URI as parameters and returns a promise resolving 
+ * to an object containing the URI of the rating entry and its details. It verifies the signature of the amended rating, 
+ * validates required fields, and generates the necessary signatures for the revised entry. This is essential 
+ * for maintaining the integrity and trustworthiness of the rating system, especially in decentralized environments.
+ *
+ * This function is specifically designed for use by Rating Authors or Ledger/API Providers.
+ * 
+ * @param rating - The rating revise entry to process.
+ *                             This includes the original rating's digest, signature, and other relevant details.
+ * @param chainSpace - The identifier of the blockchain space (as a URI) where the rating is being revoked.
```
Comment (NiranjanAP) [inline packages/network-score/src/Scoring.ts]: Should we call it revoked? or revised?

File: packages/network-score/src/Scoring.ts [hunk 2]
```diff
@@ -458,10 +455,42 @@ export async function buildFromRevokeRatingProperties(
...
+ *                            This helps in pinpointing the exact location on the blockchain where the rating resides.
+ * @param authorUri - The Decentralized Identifier (DID) URI of the author who is revoking the rating.
```
Comment (NiranjanAP) [inline packages/network-score/src/Scoring.ts]: same here - revoke or revise?

File: packages/network-score/src/Scoring.ts (hunk 1 above)
Comment (belyrs) [reply packages/network-score/src/Scoring.ts]: You are right. Revised is the correct word.

File: packages/network-score/src/Scoring.ts (hunk 2 above)
Comment (belyrs) [reply packages/network-score/src/Scoring.ts]: You are right. Revised is the correct word.

**Review by amarts — APPROVED** [review]

**Pull request merged**